doc/tutorial/introduction/introduction_align.cpp

```diff
@@ -20,26 +16,22 @@ int main()
     {
         // Create a /tmp/my.fasta file.
         seqan3::sequence_file_output file_out{filename};
-        file_out.emplace_back("ACGTGATG"_dna4, std::string{"seq1"});
-        file_out.emplace_back("AGTGATACT"_dna4, std::string{"seq2"});
+
+        using seqan3::operator""_dna5;  // for better readability of next code lines
+
+        file_out.emplace_back("ACGTGATG"_dna5, std::string{"seq1"});
+        file_out.emplace_back("AGTGATACT"_dna5, std::string{"seq2"});
     }
 
-//! [sequence_input]
-    // Initialise a file input object with a FastA file.
-    seqan3::sequence_file_input file_in{filename}; // filename: "seq.fasta"
+    // Initialise a file input object and a vector
+    seqan3::sequence_file_input file_in{filename};
+    std::vector<seqan3::dna5_vector> sequences;
 
-    // Retrieve the sequences and ids.
-    for (auto &[seq, id, qual] : file_in)
+    for (auto & [ seq, id, qual ] : file_in)
```

Comment:
If we do not use id and qual here, should we give them variable names then?

Reply:
I think its mandatory to name all existing fields: seqan3::fields<seqan3::field::seq, seqan3::field::id, seqan3::field::qual>>

Reply:
Maybe, I am wrong but I think it is possible to use _ instead of id and qual. But yeah, I am not sure if using these would not decrease readability...

Reply:
Unfortunately, this only works in python.